crates/polars-pipe/src/operators/chunks.rs

```diff
@@ -28,5 +28,152 @@ impl DataChunk {
 }
 
 pub(crate) fn chunks_to_df_unchecked(chunks: Vec<DataChunk>) -> DataFrame {
-    accumulate_dataframes_vertical_unchecked(chunks.into_iter().map(|c| c.data))
+    let mut combiner = SemicontiguousVstacker::<4194304>::new();
```

Comment:
I am not sure we must use that here. Let's first only fix the `writers`. 

For the operators, row-size is more important than size in megabytes.

Reply:
I did it here in order to speed up `OrderedSink`, I can move it up the callstack so it's more isolated to only that.

Reply:
Help me, why does this speed up `OrderdSink`? :thinking:

Reply:
Bad phrasing, sorry. It doesn't speed up OrderedSink, it speeds up writing the resulting DataFrame to parquet (the `collect(streaming=True).write_parquet()` case, `STREAMING COLLECT + WRITE` in the benchmark).